Description: Use archive/tar instead of dmcgowan/go-tar
 It seems that dmcgowan/go-tar is basically ahead of archive/tar, and its
 purpose is to make the latest tar implementation available to older version
 of go.
 .
 We don't really care since the version of go shipped right now is v1.10,
 which already provides everything needed.
Reviewed-By: Dmitry Smirnov <onlyjob@debian.org>
Author: Arnaud Rebillout <arnaud.rebillout@collabora.com>
Reviewed-By: Dmitry Smirnov <onlyjob@debian.org>
Forwarded: not-needed
Last-Update: 2018-03-01

--- a/containerd/archive/tar.go
+++ b/containerd/archive/tar.go
@@ -16,8 +16,9 @@
 
 package archive
 
 import (
+	"archive/tar"
 	"context"
 	"fmt"
 	"io"
 	"io/ioutil"
@@ -30,9 +31,8 @@
 	"time"
 
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/continuity/fs"
-	"github.com/dmcgowan/go-tar"
 	"github.com/pkg/errors"
 )
 
 var bufPool = &sync.Pool{
--- a/containerd/archive/tar_unix.go
+++ b/containerd/archive/tar_unix.go
@@ -24,9 +24,9 @@
 	"sync"
 	"syscall"
 
 	"github.com/containerd/continuity/sysx"
-	"github.com/dmcgowan/go-tar"
+	"archive/tar"
 	"github.com/opencontainers/runc/libcontainer/system"
 	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
 )
--- a/containerd/archive/strconv.go
+++ b/containerd/archive/strconv.go
@@ -22,9 +22,9 @@
 	"strconv"
 	"strings"
 	"time"
 
-	"github.com/dmcgowan/go-tar"
+	"archive/tar"
 )
 
 // Forked from https://github.com/golang/go/blob/master/src/archive/tar/strconv.go
 // as archive/tar doesn't support CreationTime, but does handle PAX time parsing,
--- a/containerd/archive/tar_windows.go
+++ b/containerd/archive/tar_windows.go
@@ -35,9 +35,9 @@
 	"github.com/Microsoft/go-winio"
 	"github.com/Microsoft/hcsshim"
 	"github.com/containerd/containerd/log"
 	"github.com/containerd/containerd/sys"
-	"github.com/dmcgowan/go-tar"
+	"archive/tar"
 )
 
 const (
 	// MSWINDOWS pax vendor extensions
